core/common/src/LocalTime.kt

```diff
@@ -40,6 +40,11 @@ public expect class LocalTime : Comparable<LocalTime> {
          */
         public fun parse(isoString: String): LocalTime
 
+        /**
+         * Returns a LocalTime with the specified [secondOfDay]. The nanosecond field will be set to zero.
+         */
+        public fun ofSecondOfDay(secondOfDay: Int): LocalTime
```

Comment:
It should be documented when and what this function throws with the `@throws` documentation block.
I presume it should be an `IllegalArgumentException`.
There should be a test checking that such exception or its inheritor is indeed thrown in case of invalid input.